cmd/server.go

```diff
@@ -98,16 +98,17 @@ const (
 	SilenceVCSStatusNoPlans    = "silence-vcs-status-no-plans"
 	SilenceAllowlistErrorsFlag = "silence-allowlist-errors"
 	// SilenceWhitelistErrorsFlag is deprecated for SilenceAllowlistErrorsFlag.
-	SilenceWhitelistErrorsFlag = "silence-whitelist-errors"
-	SkipCloneNoChanges         = "skip-clone-no-changes"
-	SlackTokenFlag             = "slack-token"
-	SSLCertFileFlag            = "ssl-cert-file"
-	SSLKeyFileFlag             = "ssl-key-file"
-	TFDownloadURLFlag          = "tf-download-url"
-	VCSStatusName              = "vcs-status-name"
-	TFEHostnameFlag            = "tfe-hostname"
-	TFETokenFlag               = "tfe-token"
-	WriteGitCredsFlag          = "write-git-creds"
+	SilenceWhitelistErrorsFlag    = "silence-whitelist-errors"
+	SkipCloneNoChanges            = "skip-clone-no-changes"
+	SlackTokenFlag                = "slack-token"
+	SSLCertFileFlag               = "ssl-cert-file"
+	SSLKeyFileFlag                = "ssl-key-file"
+	TFDownloadURLFlag             = "tf-download-url"
+	VCSStatusName                 = "vcs-status-name"
+	TFEHostnameFlag               = "tfe-hostname"
+	TFETokenFlag                  = "tfe-token"
+	WriteGitCredsFlag             = "write-git-creds"
+	LyftApplyAuditSnsTopicArnFlag = "lyft-apply-audit-sns-topic-arn"
```

Comment:
😆   This name haha.   Can we replace apply with jobs since we might add other types of jobs to this workflow?

I do like we've added lyft here though since we'll want to keep this constrained in our repo.